config/manifests/bases/azure-service-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,394 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
This a cunning way to have a default bundle with the questions answered and instructions/icons prepopulated. I'm not very experienced with kustomize so it didn't occur to me to use it to do this, thanks!